src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -670,6 +681,38 @@ install_packages_in_root (RpmOstreeTreeComposeContext  *self,
   return TRUE;
 }
 
+static gboolean
+parse_treefile_to_json (const char    *treefile_path,
+                        JsonParser   **out_parser,
+                        GError       **error)
+{
+#ifdef HAVE_RUST
+  g_autofree char *fdpath = NULL;
+  g_auto(GLnxTmpfile) json_contents = { 0, };
+  if (opt_yaml_manifest ||
+      g_str_has_suffix (treefile_path, ".yaml") ||
+      g_str_has_suffix (treefile_path, ".yml"))
```

Comment:
Let's detect this outside the conditional, and throw a clear error `#ifndef HAVE_RUST`.